### ISSUE TITLE
🐛 [BUG] Storage Path is missing

### ISSUE THREAD

**Issue by PKuhlmay** [FriendsOfTYPO3/extension_builder#738]
In some cases the storage path is missing. 

After saving:
![image](https://github.com/FriendsOfTYPO3/extension_builder/assets/107543296/cbf2b24b-cf36-4a01-adce-1e6a569664f6)

And in the footer:
![image (1)](https://github.com/FriendsOfTYPO3/extension_builder/assets/107543296/ab504996-3c25-49e7-9729-11ae51ae5d83)

**Comment by PKuhlmay** [comment]
This might occur, when there is no entry for "repositories" inside the composer.json
```
"repositories": {
        "local": {
            "type": "path",
            "url": "Packages/*"
        }
    },
```

This should solve this issue. 

Even though the extension_builder should fetch this and maybe creates an empty folder if there is nothing set?

**Comment by crookoo** [comment]
Yes, you are right. The missing "repositories" entry was the cause for this.

Concerning your question: Maybe it would be better to tell the user that he needs to add the "repository" entry in the composer.json rather than adding it automatically?

And something else: The user not only needs to add the "repositories" entry in the composer.json. One also needs to create the packages folder in the root level of the project. Otherwise the following error message is shown:

![image](https://github.com/FriendsOfTYPO3/extension_builder/assets/16896988/e4956882-d91c-4b05-b0d9-f9d560a382e2)

**Comment by PKuhlmay** [comment]
I already added a hint inside the documentation: https://docs.typo3.org/p/friendsoftypo3/extension-builder/main/en-us/Installation/Index.html

There should also be a notification in the backend, that this is missing. Something like this:
![Bildschirmfoto 2024-03-26 um 11 31 13](https://github.com/FriendsOfTYPO3/extension_builder/assets/107543296/60b8799c-8e24-410b-9ec0-eadea17e5a3d)

**Comment by PKuhlmay** [comment]
I added the notification for this, when no storage path is detected. At the moment the notification is shown twice, but this is another problem and not related to this one. I close this for now. This will be released in v12.0.0-beta.3
